osx/Scripts/build.sh

```diff
@@ -87,8 +87,8 @@ if [ -f "$helper" ]; then
   spctl --assess --verbose=4 $app_name.app
   echo "Checking Helper..."
   codesign -dvvvv $app_name.app/Contents/Library/LaunchServices/keybase.Helper
-  echo " "
-  spctl --assess --verbose=4 $app_name.app/Contents/Library/LaunchServices/keybase.Helper
+  # You don't spctl assess binaries anymore (only bundles)
```

Comment:
This seems a bit scary -- anymore as of when?  What if someone still has an OS version that does?